src/adapters/interfaces/IMorphoMarketV1AdapterV2.sol

```diff
@@ -2,35 +2,27 @@
 // Copyright (c) 2025 Morpho Association
 pragma solidity >=0.5.0;
 
-import {IAdapter} from "../../interfaces/IAdapter.sol";
+import {ITimelockedAdapter} from "../../interfaces/ITimelockedAdapter.sol";
 import {MarketParams} from "../../../lib/morpho-blue/src/interfaces/IMorpho.sol";
 
-interface IMorphoMarketV1AdapterV2 is IAdapter {
+interface IMorphoMarketV1AdapterV2 is ITimelockedAdapter {
     /* EVENTS */
```

Comment:
other benefits of no inheritance we didn't discuss:
- all events defined at the same place
- all errors defined at the same place